### PR TITLE
mgr/dashboard: dashboard help command showing wrong syntax for login-banner

### PR DESCRIPTION
**Pull request by Sarthak0702**
Fixes: https://tracker.ceph.com/issues/56137
Signed-off-by: Sarthak0702 <sarthak.dev.0702@gmail.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [x] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: src/pybind/mgr/dashboard/module.py [hunk 1]
```diff
@@ -421,18 +421,24 @@ def set_rgw_credentials(self):
...
+        Set the custom login banner read from -i <file>
+        '''
         item_label = 'login banner file'
         if inbuf is None:
             return HandleCommandResult(
                 -errno.EINVAL,
                 stderr=f'Please specify the {item_label} with "-i" option'
             )
-        set_login_banner_mgr(inbuf, mgr_id)
+        set_login_banner_mgr(inbuf)
```
Comment (sunilangadi2) [inline src/pybind/mgr/dashboard/module.py:434]: @Sarthak0702 Changes LGTM, But can add the CLI output of your local test changes?

**Review by sunilangadi2 — APPROVED** [review]

File: src/pybind/mgr/dashboard/services/custom_banner.py [hunk 2]
```diff
@@ -6,22 +6,19 @@
...
 
 
-def set_login_banner_mgr(inbuf: str, mgr_id: Optional[str] = None):
-    item_key = 'custom_login_banner'
-    if mgr_id is not None:
-        mgr.set_store(_get_localized_key(mgr_id, item_key), inbuf)
-    else:
-        mgr.set_store(item_key, inbuf)
+def set_login_banner_mgr(inbuf: str):
+    mgr.set_store(ITEM_KEY, inbuf)
```
Comment (epuertat) [inline src/pybind/mgr/dashboard/services/custom_banner.py:13]: These services look more anemic now. What about removing this file and moving this code inside the CLI commands (encapsulating service logic makes sense when it's complex, but in this case it's almost trivial: the service itself is provided by the `mgr.set|get_store`).

File: src/pybind/mgr/dashboard/module.py (hunk 1 above)
Comment (Sarthak0702) [reply src/pybind/mgr/dashboard/module.py:434]:
```
[root@ceph ceph]# ceph dashboard set-login
2022-06-21T18:18:02.888+0000 7f38caf66700 -1 WARNING: all dangerous and experimental features are enabled.
2022-06-21T18:18:02.893+0000 7f38caf66700 -1 WARNING: all dangerous and experimental features are enabled.
no valid command found; 2 closest matches:
dashboard set-login-credentials <username>
dashboard set-login-banner
Error EINVAL: invalid command
```

Comment (Sarthak0702) [reply src/pybind/mgr/dashboard/module.py:434]: and for `ceph dashboard --help` :
```
dashboard get-login-banner                                                     Get the custom login banner text
dashboard set-login-banner                                                     Set the custom login banner read from -i <file>
dashboard unset-login-banner                                                   Unset the custom login banner
```

Comment (sunilangadi2) [reply src/pybind/mgr/dashboard/module.py]: Thanks @Sarthak0702 LGTM.

**Review by epuertat — APPROVED** [review]

**Pull request merged**